### PR TITLE
Allows to run the ruby scaffold with docker

### PR DESCRIPTION
**Pull request by Malian**
:wave: @jbruggem :)

### CODE REVIEW

File: scaffolds/ruby/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM ruby:3.0
+
+# throw errors if Gemfile has been modified since Gemfile.lock
+RUN bundle config --global frozen 1
```
Comment (jbruggem) [inline scaffolds/ruby/Dockerfile:4]: I had this error when trying to build this Dockerfile:

```
Step 5/5 : RUN bundle install
 ---> Running in 638a3d385e92
/usr/local/lib/ruby/3.0.0/rubygems.rb:281:in `find_spec_for_exe': Could not find 'bundler' (1.17.1) required by your /usr/src/app/Gemfile.lock. (Gem::GemNotFoundException)
To update to the latest version installed on your system, run `bundle update --bundler`.
To install the missing version, run `gem install bundler:1.17.1`
	from /usr/local/lib/ruby/3.0.0/rubygems.rb:300:in `activate_bin_path'
	from /usr/local/bin/bundle:23:in `<main>'

```

This change fixed it :

```suggestion
RUN bundle config --global frozen 1 && gem install bundler:1.17.1
```

File: scaffolds/ruby/docker.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+ENABLE_TTY=${ENABLE_TTY:--t}
+IMAGE="code-retreat-ruby-env"
+VERSION="1"
+TAG="$IMAGE:$VERSION"
+
+docker images | grep -q "$IMAGE[ ]*$VERSION" || docker build -t $TAG .
```
Comment (jbruggem) [inline scaffolds/ruby/docker.sh:8]: Improvement suggestion: stop if building the image fails:

```suggestion
docker images | grep -q "$IMAGE[ ]*$VERSION" || docker build -t $TAG .  || { echo "Failed to build image"; exit 1; }
```

**Review by jbruggem — CHANGES_REQUESTED** [review]
Hello :wave: 

Nice job :). Thanks for the contribution !

File: scaffolds/ruby/Dockerfile (hunk 1 above)
Comment (Malian) [reply scaffolds/ruby/Dockerfile:4]: Interesting, I do not encounter this error on my computer :thinking: Let's commit your changes!

**Review by jbruggem — APPROVED** [review]

**Pull request merged**